src/components/container/sticky-bar/sticky-bar.scss

```diff
@@ -0,0 +1,56 @@
+@use '../../core/styles' as sbb;
+
+// Default component properties, defined for :host. Properties which can not
+// travel the shadow boundary are defined through this mixin
+@include sbb.host-component-properties;
+
+:host {
+  display: contents;
+}
+
+.sbb-sticky-bar {
+  position: sticky;
+  display: flex;
+  width: 100%;
+  bottom: 0;
+  background-color: var(--sbb-sticky-bar-bg-color);
+  padding-block: var(--sbb-spacing-responsive-xxxs);
+}
+
+.sbb-sticky-bar::before {
+  @include sbb.shadow-level-5-hard;
+
+  position: absolute;
+  display: inline-block;
+  content: '';
+  height: 100%;
+  width: 100vw;
```

Comment:
Discussion point:
I understand why `width: 100vw;` is needed but it prevents any possible use of a non-full page sticky bar (imagine a page where you have a side menu for example).

The best solution would be to negate the spacing of the `sbb.page-spacing; | sbb.page-spacing-expanded` but I understand this might be hard to achieve.
A good compromise could be to a CSS variable defined on the `:host` so that the consumer  has a way to handle those scenarios

Reply:
We asked Manuel about this and he was positive the sticky bar is only meant to be used in full width

Reply:
In any case, I think it is always better to anticipate the changes

Reply:
After discussing the matter with @kyubisation, we decided to try with an "infinite" sticky-bar with `overflow-x:hidden` on the sbb-container. 
But after some tests I noticed that any kind of overflow rule prevents the `position: sticky` from working. Another solution could be `contain: contents` (which works just like `overflow: hidden`), which does not cause any problems with the sticky positioning, but it messes up any overlay that goes outside the container. 
So we'll use the css variable approach